d2l-pdf-viewer.js

```diff
@@ -549,46 +555,64 @@ Polymer({
 	ready: function() {
 		this._boundListeners = false;
 		this._addedEventListeners = false;
-		// Import pdfjs separately to allow better code splitting
-		this._initializeTask = Promise.all([
-			import('pdfjs-dist-modules/pdf.js'),
-			import('pdfjs-dist-modules/pdf_link_service.js'),
-			import('pdfjs-dist-modules/pdf_viewer.js')
-		])
-			.then(([pdfImport, pdfLinkServiceImport, pdfViewerImport]) => {
-				const pdf = pdfImport.default;
-				const { LinkTarget } = pdfImport;
-				const { PDFLinkService } = pdfLinkServiceImport;
-				const { PDFViewer } = pdfViewerImport;
-
-				this._pdf = pdf;
-				// Ensure that style scoping is applied to elements added by PDF.js
-				// under Shady DOM
-				this.scopeSubtree(this.$.viewerContainer, true);
-
-				pdf.GlobalWorkerOptions.workerSrc =
-					this.pdfJsWorkerSrc || import.meta.url + '/../node_modules/pdfjs-dist-modules/pdf.worker.min.js';
-
-				// (Optionally) enable hyperlinks within PDF files.
-				this._pdfLinkService = new PDFLinkService({
-					externalLinkTarget: LinkTarget.BLANK
-				});
-
-				this._pdfViewer = new PDFViewer({
-					container: this.$.viewerContainer,
-					linkService: this._pdfLinkService,
-					useOnlyCssZoom: true, // Use CSS zooming only, as default zoom rendering in (modularized?) pdfjs-dist is buggy
-				});
-
-				this._pdfLinkService.setViewer(this._pdfViewer);
-
-				// Add event listeners before loading document
-				this._addEventListeners();
-
-				this.dispatchEvent(new CustomEvent('d2l-pdf-viewer-initialized', {
-					bubbles: true,
-				}));
-			});
+
+		const initializeTask = this.bundled
+			? Promise.all([
+				import('pdfjs-dist-modules/pdf.js'),
+				import('pdfjs-dist-modules/pdf_link_service.js'),
+				import('pdfjs-dist-modules/pdf_viewer.js')
+			]) : this._loadScript('https://cdn.jsdelivr.net/npm/pdfjs-dist@2.0.943/build/pdf.min.js')
+				.then(this._loadScript.bind(this, 'https://cdn.jsdelivr.net/npm/pdfjs-dist@2.0.943/web/pdf_viewer.js'));
+
+		this._initializeTask = initializeTask
+			.then(this._librariesLoaded.bind(this));
+	},
+	_loadScript: function(src) {
+		const scriptTag = document.createElement('script');
+		scriptTag.async = false;
+		document.head.appendChild(scriptTag);
+		return new Promise((resolve, reject) => {
+			scriptTag.onload = resolve;
+			scriptTag.onerror = reject;
+			scriptTag.src = src;
+		}).catch(() => {
+			const progressBar = this.$.progressBar;
```

Comment:
```suggestion
			this.$.progressBar.hidden = true;
```

And remove the next line, since it isn't being reused in this context (not saving anything).

Reply:
Also, can make this the catch for `_initializeTask` so it catches both import paths equally.